### DISCUSSION TITLE
Passing glb from threejs scene to modelviewer with GLTFExporter error

### DISCUSSION THREAD

**Discussion by ZackARC-solutions** [google/model-viewer#3664] [Q&A]
I'm trying to pass a glb created in a threejs scene to webxr / quicklook via GLTFExporter.
 
  let modelViewer = document.getElementById("ar-viewer");
  exporter = new THREE.GLTFExporter();
    exporter.parse(
      group,
      function (gltf) {
        var Url = saveArrayBuffer(gltf, "scene.glb");
        modelViewer.setAttribute("src", Url);
        setTimeout(() => {
          URL.revokeObjectURL(Url);
        }, 2000);
      },
      { binary: true }
    );

This has worked fine in the past, however, I now am receiving a 
WebKitBlobResource error 1 on iOS (TypeError: Load failed)
and
TypeError: Failed to fetch

Is anyone else receiving these errors?

**Comment by allenwixted** [comment]
> I'm trying to pass a glb created in a threejs scene to webxr / quicklook via GLTFExporter.
> 
>  
> 
>   let modelViewer = document.getElementById("ar-viewer");
> 
>   exporter = new THREE.GLTFExporter();
> 
>     exporter.parse(
> 
>       group,
> 
>       function (gltf) {
> 
>         var Url = saveArrayBuffer(gltf, "scene.glb");
> 
>         modelViewer.setAttribute("src", Url);
> 
>         setTimeout(() => {
> 
>           URL.revokeObjectURL(Url);
> 
>         }, 2000);
> 
>       },
> 
>       { binary: true }
> 
>     );
> 
> 
> 
> This has worked fine in the past, however, I now am receiving a 
> 
> WebKitBlobResource error 1 on iOS (TypeError: Load failed)
> 
> and
> 
> TypeError: Failed to fetch
> 
> 
> 
> Is anyone else receiving these errors? 
> 
> 
> 
> 
> 
> 

Have you found a solution?
